### DISCUSSION TITLE
I like the new "skin" but the hitsounds are awful (make them changable)

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [ppy/osu#28061] [Ideas] [answered]
Currently if you wanna change the hitsounds, you have to import osu classic skins. What I tied is removing everything except sounds but it will still use the classic skin as long as you use any piece of classic skin.

Make hitsounds editable or at least come up with better ones. A clicky ones like in avenir skin or similar classic skins would be nicer and more in line with the rest of the UI sounds. In main menu you have nice clicky sounds when hovering and clicking.

**Comment by peppy** [comment, marked as answer]
Try argon "pro". It has more "clicky" hit sounds if you want that kinda shiz.

**Reply by SolsticeSpectrum** [reply]
Oh yeah pro sounds much better, btw will it ever be possible to change argon? I know it works differently, it doesn't use textures, it's all coded but still, it would be nice if it was allowed to change the code through skin or something like that. Like a scripting language for skins maybe? Currently if you export argon skin, you get couple of config files but iirc you can't even change colors.

**Reply by peppy** [reply]
Eventually it may be possible, yes.

**Reply by SolsticeSpectrum** [reply]
For the time being, could we at least get black argon? Or a small toggle in menu to disable colors
